src/sys/mod.rs

```diff
@@ -88,3 +88,6 @@ pub mod uio;
 pub mod utsname;
 
 pub mod wait;
+
+#[cfg(target_os = "linux")]
```

Comment:
Any reason not to include Android, too?

Reply:
No, I was concerned that android might not have some of the flags introduced in 4.17 (IN_MASK_CREATE and IN_MASK_ADD) but the libc doesn't either.